components/ProfileMenu.vue

```diff
@@ -34,64 +40,94 @@ const logout = async () => {
-                />
-              </Switch>
+                  class="relative inline-flex h-6 w-10 shrink-0 cursor-pointer rounded-full p-1 transition-colors duration-200 ease-in-out focus:outline-none"
+                >
+                  <span
+                    aria-hidden="true"
+                    :class="
+                      profileStore.autoplay
+                        ? 'translate-x-full'
+                        : 'translate-x-0'
+                    "
+                    class="pointer-events-none inline-block aspect-square w-4 transform rounded-full bg-white-1 shadow-lg ring-1 ring-black-separator transition duration-200 ease-in-out"
+                  />
+                </Switch>
+              </button>
             </MenuItem>
-            <MenuItem v-slot="{ active }" as="div">
+            <MenuItem v-slot="{ active }">
               <button
-                :class="active ? 'bg-violet-500 text-white' : 'text-gray-900'"
-                class="w-full px-2 py-2 text-left hover:bg-[red]"
+                :class="{
+                  'bg-label-separator dark:bg-label-dark-separator': active,
+                }"
+                class="w-full rounded-lg px-3 py-2 text-left"
                 @click.stop
               >
-                {{ $t("profile.theme") }}
+                <p>{{ $t("profile.theme") }}</p>
+                <span class="text-label-2 dark:text-label-dark-2">
+                  {{ $t(`profile.theme-${$colorMode.value}`) }}
+                </span>
               </button>
             </MenuItem>
             <MenuItem v-slot="{ active }" as="div">
               <button
-                :class="active ? 'bg-violet-500 text-white' : 'text-gray-900'"
-                class="w-full px-2 py-2 text-left hover:bg-[red]"
+                :class="{
+                  'bg-label-separator dark:bg-label-dark-separator': active,
+                }"
+                class="w-full rounded-lg px-3 py-2 text-left"
                 @click.stop
               >
-                {{ $t("profile.app-language") }}
+                <p>{{ $t("profile.app-language") }}</p>
+                <span class="text-label-2 dark:text-label-dark-2">{{
+                  $i18n.locale
+                }}</span>
               </button>
             </MenuItem>
             <MenuItem v-slot="{ active }" as="div">
               <button
-                :class="active ? 'bg-violet-500 text-white' : 'text-gray-900'"
-                class="w-full px-2 py-2 text-left hover:bg-[red]"
+                :class="{
+                  'bg-label-separator dark:bg-label-dark-separator': active,
+                }"
+                class="w-full rounded-lg px-3 py-2 text-left"
                 @click.stop
               >
-                {{ $t("profile.content-language") }}
+                <p>{{ $t("profile.content-language") }}</p>
+                <span class="text-label-2 dark:text-label-dark-2">{{
+                  $i18n.locale
```

Comment:
The value displayed here has nothing to do with the content-language. Please fix by either removing this value or replacing it with the correct value.

Reply:
I think I display the correct values now, but the switching functionality should be implemented in a separate PR